### PR TITLE
PLFM-5170-VI rolling back initial approach, applied to 33 controllers

### PR DESCRIPTION
**Pull request by brucehoff**

### CODE REVIEW

File: services/authutil/src/main/java/org/sagebionetworks/authutil/SimpleCORSFilter.java [hunk 1]
```diff
@@ -31,7 +30,9 @@
 	public static final String ACCESS_CONTROL_REQUEST_METHOD = "Access-Control-Request-Method";
 	public static final String ALL_ORIGINS = "*";
 	public static final String METHODS = "POST, GET, PUT, DELETE";
-	public static final String HEADERS = "Origin, X-Requested-With, Content-Type, Accept, " + AuthorizationConstants.SESSION_TOKEN_PARAM;
+	public static final String HEADERS = "Origin, X-Requested-With, Content-Type, Accept, " + 
+			AuthorizationConstants.SESSION_TOKEN_PARAM +
+		", "+AuthorizationConstants.AUTHORIZATION_HEADER_NAME;
```
Comment (marcomarasca) [inline services/authutil/src/main/java/org/sagebionetworks/authutil/SimpleCORSFilter.java:35]: Double check that if the Auth header is considered a "custom" header?

File: services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/oauth/OpenIDConnectManagerImpl.java [hunk 2]
```diff
@@ -360,52 +361,20 @@ public OIDCTokenResponse getAccessToken(String code, String verifiedClientId, St
...
-		return result;
-	}	
-	
-	@Override
-	public Object getUserInfo(UserInfo userAuthorization, String oauthClientId, String oauthEndpoint) {
-		Long userId = userAuthorization.getId();
-		Date authTime = authDao.getSessionValidatedOn(userId);
 
-		Map<OIDCClaimName,Object> userInfo = getUserInfo(userId.toString(), userAuthorization.getScopes(), userAuthorization.getOidcClaims());
+		Map<OIDCClaimName,Object> userInfo = getUserInfo(userId.toString(), scopes, oidcClaims);
```
Comment (marcomarasca) [inline services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/oauth/OpenIDConnectManagerImpl.java:377]: userId.toString() not needed as it's already a String

File: services/repository/src/main/java/org/sagebionetworks/repo/web/filter/JsonpCallbackFilter.java [hunk 3]
```diff
@@ -47,6 +47,10 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 			if(sessionToken != null){
 				throw new ServletException("JSONP callbacks are only allowed on public methods with no 'sessionToken'");
 			}
+			String accessToken = httpRequest.getHeader(AuthorizationConstants.AUTHORIZATION_HEADER_NAME);
```
Comment (marcomarasca) [inline services/repository/src/main/java/org/sagebionetworks/repo/web/filter/JsonpCallbackFilter.java:50]: Double check with Jay that this does not break the web client

**Review by marcomarasca — APPROVED** [review]

File: services/authutil/src/main/java/org/sagebionetworks/authutil/SimpleCORSFilter.java (hunk 1 above)
Comment (brucehoff) [reply services/authutil/src/main/java/org/sagebionetworks/authutil/SimpleCORSFilter.java:35]: Per Jay, this is correct.

File: services/repository/src/main/java/org/sagebionetworks/repo/web/filter/JsonpCallbackFilter.java (hunk 3 above)
Comment (brucehoff) [reply services/repository/src/main/java/org/sagebionetworks/repo/web/filter/JsonpCallbackFilter.java:50]: Per Jay, this feature is most likely no longer used and therefore the change is irrelevant.

**Pull request merged**